adapters/Mintegral/MintegralAdapter/GADMediationAdapterMintegral.m

```diff
@@ -4,44 +4,105 @@
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
 //
-//     http://www.apache.org/licenses/LICENSE-2.0
+//     https://www.apache.org/licenses/LICENSE-2.0
 //
 // Unless required by applicable law or agreed to in writing, software
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-//
 
 #import "GADMediationAdapterMintegral.h"
-
 #import "GADMediationAdapterMintegralConstants.h"
+#import "GADMAdapterMintegralExtras.h"
+#import "GADMAdapterMintegralRewardedAdRenderer.h"
+#import "GADMAdapterMintegralBannerRenderer.h"
+#import "GADMAdapterMintegralInterstitialRenderer.h"
+#import "GADMAdapterMintegralNativeRenderer.h"
+#import "GADMAdapterMintegralUtils.h"
+#import <MTGSDK/MTGSDK.h>
+#import <MTGSDKBidding/MTGBiddingSDK.h>
+
+
```

Comment:
I don't see any methods related to COPPA, CCPA and GDPR, don't we need to handle them as well? According to Karen, CCPA and GDPR values need to be passed to the Mintegral SDK before it initializes.

Reply:
@Mintegral-Lee Gentle bump on this, this is a blocking comment.

Reply:
Thanks! Just to confirm, does publisher need to add other lines of code instead of `[[MTGSDK sharedInstance] setConsentStatus:YES/NO];` or `[[MTGSDK sharedInstance] setDoNotTrackStatus:YES/NO];`?


In Android we need to do the below:
```
MBridgeSDK sdk = MBridgeSDKFactory.getMBridgeSDK();
Map<String, String> map = sdk.getMBConfigurationMap("your appId", "your appKey");
sdk.setConsentStatus(MainApplication.this, MBridgeConstans.IS_SWITCH_ON);
```

Reply:
Also, does Mintegral SDK support COPPA?

Reply:
> Also, does Mintegral SDK support COPPA?

Mintegral ad sdk is not provided, but publishers can configure it in the Mintegral potal.

Reply:
> Thanks! Just to confirm, does publisher need to add other lines of code instead of just `[[MTGSDK sharedInstance] setConsentStatus:YES/NO];` or `[[MTGSDK sharedInstance] setDoNotTrackStatus:YES/NO];`?
> 
> In Android we need to do the below:
> 
> ```
> MBridgeSDK sdk = MBridgeSDKFactory.getMBridgeSDK();
> Map<String, String> map = sdk.getMBConfigurationMap("your appId", "your appKey");
> sdk.setConsentStatus(MainApplication.this, MBridgeConstans.IS_SWITCH_ON);
> ```

No need to add other lines of code.

Reply:
Thanks for confirming!